ci-operator/jobs/openshift/origin/openshift-origin-master-presubmits.yaml

```diff
@@ -425,6 +425,149 @@ presubmits:
+        - mountPath: /usr/local/e2e-aws-multitenant-cluster-profile
+          name: cluster-profile
+        - mountPath: /usr/local/e2e-aws-multitenant
+          name: job-definition
+          subPath: cluster-launch-installer-e2e.yaml
+        - mountPath: /etc/sentry-dsn
+          name: sentry-dsn
+          readOnly: true
+      serviceAccountName: ci-operator
+      volumes:
+      - name: cluster-profile
+        projected:
+          sources:
+          - secret:
+              name: cluster-secrets-aws
+      - configMap:
+          name: prow-job-cluster-launch-installer-e2e
+        name: job-definition
+      - name: sentry-dsn
+        secret:
+          secretName: sentry-dsn
+    trigger: (?m)^/test( | .* )e2e-aws-multitenant,?($|\s.*)
+  - agent: kubernetes
+    always_run: false
+    branches:
+    - master
+    context: ci/prow/e2e-aws-ovn-kubernetes
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-origin-master-e2e-aws-ovn-kubernetes
+    optional: true
+    rerun_command: /test e2e-aws-ovn-kubernetes
+    spec:
+      containers:
+      - args:
+        - --artifact-dir=$(ARTIFACTS)
+        - --give-pr-author-access-to-namespace=true
+        - --secret-dir=/usr/local/e2e-aws-ovn-kubernetes-cluster-profile
+        - --sentry-dsn-path=/etc/sentry-dsn/ci-operator
+        - --target=e2e-aws-ovn-kubernetes
+        - --template=/usr/local/e2e-aws-ovn-kubernetes
+        command:
+        - ci-operator
+        env:
+        - name: CLUSTER_NETWORK_TYPE
+          value: OVNKubernetes
+        - name: CLUSTER_TYPE
+          value: aws
+        - name: CONFIG_SPEC
+          valueFrom:
+            configMapKeyRef:
+              key: openshift-origin-master.yaml
+              name: ci-operator-master-configs
+        - name: JOB_NAME_SAFE
+          value: e2e-aws-ovn-kubernetes
+        - name: TEST_COMMAND
+          value: TEST_SUITE=openshift/conformance/parallel run-tests
+        image: ci-operator:latest
+        imagePullPolicy: Always
+        name: ""
+        resources:
+          requests:
+            cpu: 10m
+        volumeMounts:
+        - mountPath: /usr/local/e2e-aws-ovn-kubernetes-cluster-profile
+          name: cluster-profile
+        - mountPath: /usr/local/e2e-aws-ovn-kubernetes
+          name: job-definition
+          subPath: cluster-launch-installer-e2e.yaml
+        - mountPath: /etc/sentry-dsn
+          name: sentry-dsn
+          readOnly: true
+      serviceAccountName: ci-operator
+      volumes:
+      - name: cluster-profile
+        projected:
+          sources:
+          - secret:
+              name: cluster-secrets-aws
+      - configMap:
+          name: prow-job-cluster-launch-installer-e2e
+        name: job-definition
+      - name: sentry-dsn
+        secret:
+          secretName: sentry-dsn
+    trigger: (?m)^/test( | .* )e2e-aws-ovn-kubernetes,?($|\s.*)
```

Comment:
Drop -kubernetes from the end, -ovn is unique across our project

Reply:
Referring to the ovn-kubernetes plugin as "ovn" is an anti-pattern and should be stamped out wherever it appears. "OVN" and "ovn-kubernetes" are two separate projects and the network team, at least, needs to be able to talk about both of them unambiguously. (We will also likely need to talk about both ovn-kubernetes and the OpenStack OVN plugin at the same time in OCP-on-OSP in the future.)

Reply:
None of our users or customers will EVER understand that distinction.

Reply:
Mike Barrett will never go into a sale and say "OVN Kubernetes".  We're offering users "OVN in OpenShift".  What other kind of OVN will there be?